source/common/config/new_grpc_mux_impl.cc

```diff
@@ -153,13 +155,36 @@ GrpcMuxWatchPtr NewGrpcMuxImpl::addWatch(const std::string& type_url,
 // subscription will enqueue and attempt to send an appropriate discovery request.
 void NewGrpcMuxImpl::updateWatch(const std::string& type_url, Watch* watch,
                                  const std::set<std::string>& resources,
-                                 bool creating_namespace_watch) {
+                                 const bool creating_namespace_watch) {
   ASSERT(watch != nullptr);
   auto sub = subscriptions_.find(type_url);
   RELEASE_ASSERT(sub != subscriptions_.end(),
                  fmt::format("Watch of {} has no subscription to update.", type_url));
-  auto added_removed = sub->second->watch_map_.updateWatchInterest(watch, resources);
-  if (creating_namespace_watch) {
+  // If this is a glob collection subscription, we need to compute actual context parameters.
+  std::set<std::string> xdstp_resources;
+  // TODO(htuch): add support for resources beyond LDS glob collections, the constraints below
+  // around resource size and ID reflect the progress of the xdstp:// implementation.
```

Comment:
What about this is specific to LDS? Is it just that LDS only has a single resource size that might be a glob? Clarify slightly?

Reply:
Updated wording, there is nothing LDS specific, it applies to CDS as well. The only collections we model explicitly today are LDS/CDS. In the future, LEDS will also be a collection type.